logic level, levelCore, coreFighter

Janakanty opened the pull request.

Janakanty committed:
logic level, levelCore, coreFighter

JezSonic committed:
Merge branch 'developer' into Torba_developer

JezSonic commented:
Miałeś konflikt  w pliku `project.godot` ale rozwiązałem go

Yazir commented on world/levels/test level/TestMap.gd:
```diff
@@ -0,0 +1,16 @@
+	
+func load_fighters_position(): #share positions for fighters
+	var fighter_node = "Fighter" 
+	for i in Global.fighters_checked_number:
```
i would advise against using singletons

Janakanty committed:
working camera frm kernik

Janakanty committed:
t push
Merge branch 'Torba_developer' of
https://github.com/NewDEV-github/godocik into Torba_developer # Please
enter a commit message to explain why this merge is necessary,

ZZZthe commit.

JezSonic committed:
Fix missing '}'
